src/main/java/com/hemant/jlambda/App.java

```diff
@@ -38,12 +43,13 @@
 
     public static void main(String[] args) {
         int exitCode =  new CommandLine(new App()).setCaseInsensitiveEnumValuesAllowed(true).execute(args);
+        logger.debug("Exiting application with the exit code: {}",exitCode);
         System.exit(exitCode);
     }
 
     @Override
     public void run() {
-
+        logger.debug("Started running application.");
```

Comment:
This log should give the intent of the application at various points in its lifecycle.
Something like
```
        logger.debug("Currently running {}", arg);
```
Here arg should be what command the user had given , that can be obtained from `args` param in main() method